src/proto/NEAR.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package TW.NEAR.Proto;
+option java_package = "wallet.core.jni.proto";
+
+/// Provides container for unsigned 128 bit integers.
+message Uint128 {
```

Comment:
No need to create a new type, just use bytes, and you can load it from wallet core (`uint128_t` in `boost`)